submit.py

```diff
@@ -1,15 +1,27 @@
 #!/usr/bin/env python
-import ConfigParser
+from __future__ import print_function
 import optparse
 import os
 import sys
 import itertools
-import mimetools
 import mimetypes
-import urllib
-import urllib2
 import traceback
-
+import random
+import string
+if sys.version_info[0] >= 3:
+    _PYTHON_VERSION = 3
+    import configparser
+    import urllib.parse
+    import urllib.request
+    import urllib.error
+else:
+    # Python 2, import modules with Python 3 names
+    _PYTHON_VERSION = 2
+    import ConfigParser as configparser
+    import urllib
+    import urllib2
+    urllib.request = urllib.error = urllib2
```

Comment:
@mikaelgoldmann suggestion: merge this as is for now since it works and is tested, then we can discuss a bit more and if desired easily change according to your suggestion separately.